cmd/flow-aggregator/flow-aggregator.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	aggregator "github.com/vmware-tanzu/antrea/pkg/flowaggregator"
```

Comment:
Please use the convention followed in Antrea.
Can you request Antonin to enable the github workflows (golang linters, spell check, unit test etc.) for feature/flow-aggregator branch?

Reply:
Thanks, done.